usaspending_api/api_contracts/contracts/search/SpendingByTransaction.md

```diff
@@ -0,0 +1,222 @@
+                        "internal_id": "68856340",
+                        "Action Date": "2018-05-21",
+                        "Recipient Name": "LEIDOS INNOVATIONS CORPORATION",
+                        "Mod": "P00206",
+                        "Award ID": "DTFAWA05C00031R",
+                        "Awarding Sub Agency": "Federal Aviation Administration"
+                    },
+                    {
+                        "Transaction Amount": "30000000.00",
+                        "Award Type": "INDEFINITE DELIVERY / INDEFINITE QUANTITY",
+                        "Awarding Agency": "Department of Transportation",
+                        "internal_id": "68856340",
+                        "Action Date": "2018-09-26",
+                        "Recipient Name": "LEIDOS INNOVATIONS CORPORATION",
+                        "Mod": "P00216",
+                        "Award ID": "DTFAWA05C00031R",
+                        "Awarding Sub Agency": "Federal Aviation Administration"
+                    },
+                    {
+                        "Transaction Amount": "9920443.99",
+                        "Award Type": "INDEFINITE DELIVERY / INDEFINITE QUANTITY",
+                        "Awarding Agency": "Department of Transportation",
+                        "internal_id": "68856340",
+                        "Action Date": "2018-09-19",
+                        "Recipient Name": "LEIDOS INNOVATIONS CORPORATION",
+                        "Mod": "P00215",
+                        "Award ID": "DTFAWA05C00031R",
+                        "Awarding Sub Agency": "Federal Aviation Administration"
+                    },
+                    {
+                        "Transaction Amount": "6384553.00",
+                        "Award Type": "INDEFINITE DELIVERY / INDEFINITE QUANTITY",
+                        "Awarding Agency": "Department of Transportation",
+                        "internal_id": "68856340",
+                        "Action Date": "2018-05-01",
+                        "Recipient Name": "LEIDOS INNOVATIONS CORPORATION",
+                        "Mod": "P00205",
+                        "Award ID": "DTFAWA05C00031R",
+                        "Awarding Sub Agency": "Federal Aviation Administration"
+                    },
+                    {
+                        "Transaction Amount": "5052922.00",
+                        "Award Type": "INDEFINITE DELIVERY / INDEFINITE QUANTITY",
+                        "Awarding Agency": "Department of Transportation",
+                        "internal_id": "69009914",
+                        "Action Date": "2018-04-24",
+                        "Recipient Name": "LEIDOS INNOVATIONS CORPORATION",
+                        "Mod": "P00120",
+                        "Award ID": "DTFAWA03C00059R",
+                        "Awarding Sub Agency": "Federal Aviation Administration"
+                    }
+                ]
```

Comment:
It is public, but do we have any concerns highlighting specific real awards in our documentation?

Reply:
Sometimes we do, usually we do not have concerns. At the moment there is no set policy or standard.
I'm often more careful if it is a single record and not concerned with list of summary records.